contrib/inventory/gce_googleapiclient.py

```diff
@@ -159,24 +169,48 @@ def get_inventory(instances):
 
 def main(args):
     project = args['--project']
+    all_projects = args['--all-projects']
     zone = args['--zone']
     api_version = args['--api-version']
-    billing_account_name = args['--account']
+    billing_account_name = args['--billing-account']
 
+    projects_list = []
+    zones_list = []
     instances = []
-    for project in get_all_billing_projects(billing_account_name):
-        for zone in get_all_zones_in_project(project):
-            for instance in get_instances(project_id=project,
-                              zone=zone,
-                              api_version=api_version):
-                instances.append(instance)
+
+    if project:
+        if type(project) is list:
```

Comment:
you don't need to check this. now that you are using `--project PROJECT...` doctopts will always return an array when getting `ARGS['--project']`